src/main/java/org/beanpod/switchboard/dto/LogDto.java

```diff
@@ -23,4 +26,16 @@
   private String level;
 
   private String serialNumber;
+
+  private StreamLog streamLog;
+
+  // for backward compatibility
+  public LogDto(
+      Long id, OffsetDateTime dateTime, String message, String level, String serialNumber) {
+    this.id = id;
+    this.dateTime = dateTime;
+    this.message = message;
+    this.level = level;
+    this.serialNumber = serialNumber;
+  }
```

Comment:
whats this for

Reply:
I believe it's used for creating logs that are not stream-related. If this constructor is commented out, a funky behavior will appear whenever logs are created